### PR TITLE
add several recent[ish] UBC faculty

### PR DESCRIPTION
**Pull request by djsutherland**
Adds some 2021 arrivals to UBC (all tenure-track or tenured research faculty in the CS department; also see https://www.cs.ubc.ca/people/faculty).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.   [not relevant]

- [x] If the institution you are adding is not in the US, update `country-info.csv`.   [not relevant]

### CODE REVIEW

File: csrankings-j.csv [hunk 1]
```diff
@@ -603,7 +603,7 @@ Jefersson A. dos Santos,UFMG,http://homepages.dcc.ufmg.br/~jefersson,wXQnnTUAAAA
 Jefersson Alex dos Santos,UFMG,http://homepages.dcc.ufmg.br/~jefersson,wXQnnTUAAAAJ
 Jeff A. Bilmes,University of Washington,https://people.ece.uw.edu/bilmes,L9QufAsAAAAJ
 Jeff Carver 0001,The University of Alabama,http://carver.cs.ua.edu,qGe7nJ0AAAAJ
-Jeff Clune,University of Wyoming,http://www.uwyo.edu/profiles/extras/playing-with-robots.html,5TZ7f5wAAAAJ
+Jeff Clune,University of British Columbia,https://www.cs.ubc.ca/people/jeff-clune,5TZ7f5wAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv:617]: Please clarify if this person's position is >75% at UBC (vs. OpenAI).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-j.csv (hunk 1 above)
Comment (djsutherland) [reply csrankings-j.csv:617]: I'm pretty sure yes, but @jeffclune can confirm.

Comment (jeffclune) [reply csrankings-j.csv:617]: Yes. I confirm I meet that threshold. Thanks!

**Comment by djsutherland** [comment]
(Rebased to resolve a merge conflict that happened in the meantime.)

**Pull request merged**